aws-android-sdk-connect/src/main/java/com/amazonaws/services/connect/model/UpdateContactFlowMetadataRequest.java

```diff
@@ -49,7 +49,7 @@
 
     /**
      * <p>
-     * The name of the contact flow.
+     * TThe name of the contact flow.
```

Comment:
Typo

Reply:
We usually don't make changes to these PRs since they are auto generated.